safe_cli/safe_completer_constants.py

```diff
@@ -160,6 +161,9 @@
     "load_ledger_cli_owners": HTML(
         "Command <b>load_ledger_cli_owners</b> show a list of ledger addresses to choose between them "
     ),
+    "load_trezor_cli_owners": HTML(
+        "Command <b>load_trezor_cli_owners</b> show a list of trezor addresses to choose between them "
```

Comment:
```suggestion
        "Command <b>load_trezor_cli_owners</b> show a list of Trezor hardware wallet addresses to choose between them "
```

Make it clear also for the ledger command